src/JsonPointer/JsonPointer.cs

```diff
@@ -205,6 +205,9 @@ public static bool TryParse(string source, [NotNullWhen(true)] out JsonPointer?
 	/// <remarks>This method creates un-encoded pointers only.</remarks>
 	public static JsonPointer Create(params PointerSegment[] segments)
 	{
+#if NET9_0_OR_GREATER
+		return Create(segments.AsSpan());
```

Comment:
This doesn't make sense to me.  You're not saving anything because you've already allocated the array that was passed in.  You don't gain anything by calling the span version here.

Reply:
You are right, it doesn't give me any benefit in terms of runtime/allocations, but instead of having 2 versions of basically the same method I can save in terms of code size, JIT time and code maintenance. That's the main reason I did it, since the AsSpan() is virtually free here.